resources/compass/charts/director/values.yaml

```diff
@@ -10,6 +10,14 @@ deployment:
   dbPool:
     maxOpenConnections: 30
     maxIdleConnections: 2
+  liveness:
```

Comment:
Please remove liveness: and readiness:, I think those should have been added by a faulty merge from upstream.

Reply:
cleaned up